src/core/log.js

```diff
@@ -163,13 +163,13 @@ export default class Log {
         this.d = console.debug;
         this.debug = console.debug;
       case "timer":
-        this.lapTime = console.timeLog;
+        this.lapTime = console.timeLog || stub();
```

Comment:
Pls add a comment that these are unavailable only on Fastly?

Reply:
added in [`6355937` (#111)](https://github.com/serverless-dns/serverless-dns/pull/111/commits/6355937702c31d4bec31f8fe85673a0fee67c2fd)